mmv1/tools/doc-parser/package-lock.json

```diff
@@ -0,0 +1,1082 @@
+{
```

Comment:
I'm not super familiar with node development, do we need to commit the lock file?

Reply:
yeah, lock is tell what dependencies to use. It ensure if someone runs the tool they are using the same versions. Similar to `go.sum`